test/OpenTelemetry.Tests/Trace/TracerProviderSdkTest.cs

```diff
@@ -222,6 +222,38 @@ public void TracerProviderSdkSamplerAttributesAreAppliedToActivity(SamplingDecis
             }
         }
 
+        [Fact(Skip = "Disabled")]
```

Comment:
maybe create an issue for this, and link to it from the Skip message, to make it easy to track.